### ISSUE TITLE
persistentvolume-binder disabled by default

### ISSUE THREAD

**Issue by SitoCH** [Project31/ansible-kubernetes-openshift-pi3#21]
Hi,
is there a reason why the persistentvolume-binder is disabled by default?

*roles/kubernetes/templates/kubeadm.yml:*
```
controllers: "*,-persistentvolume-binder,bootstrapsigner,tokencleaner"
```

In my fork I'm using GlusterFS and I had to enable the persistentvolume-binder in order to bind PersistentVolumeClaims to PersistentVolumes. So I far I didn't have issues and my pod with MySQL is bound correctly.

Thank you

Sito

**Comment by rhuss** [comment]
Actually, I copied it over from https://github.com/luxas/kubeadm-workshop at that time without when I switched to kubeadm. As it seems in the current version it's not used anymore, so I'm perfectly fine with removing it. 

Also, when time permits, I'm going to upgrade to the latest and greatest version of @luxas workshop.

**Comment by luxas** [comment]
@rhuss I'm updating kubeadm-workshop to be ready for v1.7, will be ready any day after v1.7.0 is out.

There's no need to disable that controller anymore; I disabled it in the main controller-manager but enabled it in an extra controller-manager on top of Kubernetes using a different base image. Why?

Because I used `rbd` for Dynamic Storage Provisioning and rbd shells out to the binary with the same name and the fact the vanilla controller-manager image is based on busybox with no rbd, I had to workaround it that way.

Stay tuned and oing me if I forget to report more here :)
